src/Domain.Linnapps/Reports/StockTriggerLevelReportService.cs

```diff
@@ -0,0 +1,131 @@
+﻿namespace Linn.Products.Domain.Linnapps.Reports
+{
+    using System;
+    using System.Data;
+
+    using Linn.Common.Configuration;
+    using Linn.Common.Reporting.Models;
+
+    using Oracle.ManagedDataAccess.Client;
+
+    public class StockTriggerLevelReportService : IStockTriggerLevelReportService
+    { 
+        private readonly string host = ConfigurationManager.Configuration["DATABASE_HOST"];
```

Comment:
Oh yeah. I see why it's there. 
Seems like this could be hidden in a service or a proxy service?

Reply:
Could help with testing too if the actual sql bit is encapsulated out of here